src/inference/tests/functional/ov_register_plugin_test.cpp

```diff
@@ -53,6 +53,8 @@ TEST(RegisterPluginTests, getVersionforRegisteredPluginThrows) {
 TEST(RegisterPluginTests, getVersionforNoRegisteredPluginNoThrows) {
     ov::Core core;
     ASSERT_NO_THROW(core.get_versions("unkown_device"));
+    std::map<std::string, ov::Version> versions = core.get_versions("unkown_device");
+    ASSERT_TRUE(versions.empty());
```

Comment:
```suggestion
    std::map<std::string, ov::Version> versions;
    ASSERT_NO_THROW(versions = core.get_versions("unkown_device"));
    ASSERT_TRUE(versions.empty());
```

Could you optimize number of `get_versions()` calls?

Reply:
Thank you for your review. I fixed it as per your suggestion.